tests/acceptance/auto-register-test.js

```diff
@@ -19,6 +19,13 @@ module('Acceptance | Auto register', (hooks) => {
   test('its registration is made automatically', async function (assert) {
     swInitializer.initialize(this.owner);
 
+    const promise = new Promise((resolve) => {
```

Comment:
I don't understand the need of this timeout.  The visit helper should handle the timeout.